lib/nolibc/Config.uk

```diff
@@ -2,7 +2,7 @@ menuconfig LIBNOLIBC
 	bool "nolibc: Only neccessary subset of libc functionality"
 	depends on !HAVE_LIBC
 	default y if !HAVE_LIBC
-	imply LIBUKTIME
```

Comment:
I think you can completely remove this `imply`. If we can compile without `posix-time`, we should not enable it for the build.

Reply:
Removed